showcases/angular-showcase/src/app/components/form/form.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="form-container">
+	<fieldset>
+		<form>
```

Comment:
```suggestion
	<form>
		<fieldset>
```

forms needs to surround `fieldset` elements and forms even also need to contain a `button` (at least we want to provide "good examples".